lapis2-docs/tests/docs.spec.ts

```diff
@@ -78,8 +79,6 @@ async function clickOnAllNextButtonsAndRelativeLinks(page: Page) {
     for (const pageName of pagesAfterIntroduction) {
         await page.getByRole('link', { name: `Next ${pageName}` }).click();
         await expect(page).toHaveTitle(new RegExp(`^${pageName}`));
-
-        await clickOnAllRelativeLinksInMainBody(page);
```

Comment:
Where else do we put it then? Or is there a separate issue to bring it back?

Reply:
Only deleted for my tests and I forgot it :see_no_evil: